src/components/ManagerPanelPage/ManagerPanelPage.jsx

```diff
@@ -0,0 +1,78 @@
+import { Box, Collapse, Drawer, List, ListItem, ListItemButton, ListItemText } from '@mui/material';
+import ExpandLess from '@mui/icons-material/ExpandLess';
+import ExpandMore from '@mui/icons-material/ExpandMore';
+
+import './ManagerPanelPage.css';
+
+import ListSubheader from '@mui/material/ListSubheader';
+
+function ManagerPanelPage() {
+  const [open, setOpen] = React.useState(true);
```

Comment:
You can 'import { useState } from "React"' and then use it without React. before it